kbc_transformation/transformation.py

```diff
@@ -84,15 +84,15 @@ def install_packages(packages):
         import sys
         for package in packages:
             args = [
-                sys.executable,
+                os.environ['VIRTUAL_ENV'] + '/bin/python',
                 '-m', 'pip', 'install',
                 '--disable-pip-version-check',
                 '--no-cache-dir',
                 '--no-warn-script-location', # ignore error: installed in '/var/www/.local/bin' which is not on PATH.
                 '--force-reinstall',
                 package
             ]
-            if subprocess.call(args) != 0:
+            if subprocess.call(args, stderr=sys.stdout.buffer) != 0:
```

Comment:
what's the purpose of redirecing the stderr to stdout here?

Reply:
So that "finally" the pip compatibility warnings don't get spammed to the stderr and "hide" the real transformation error.  This will prevent > 90% of transformation tickets coming to L2 :-)